File: scala/spark/hiveContext.scala
import org.apache.spark.sql.hive.HiveContext

val hiveCtx = new org.apache.spark.sql.hive.HiveContext(sc) 
val rows = hiveCtx.sql("SELECT name, age FROM users")

val firstRow = rows.first() 
println(firstRow.getString(0)) // Field 0 is the name

File: language/java/spark/hiveContext.java
import org.apache.spark.sql.hive.HiveContext; 
import org.apache.spark.sql.Row;
import org.apache.spark.sql.SchemaRDD;

HiveContext hiveCtx = new HiveContext(sc);
SchemaRDD rows = hiveCtx.sql("SELECT name, age FROM users");
Row firstRow = rows.first(); 
System.out.println(firstRow.getString(0)); // Field 0 is the name